src/adb/logcat.js

```diff
@@ -0,0 +1,10 @@
+let PassThrough = require('stream').PassThrough;
+let spawn = require('child_process').spawn;
+
+module.exports = function logcat() {
+  let proc = spawn('adb', ['logcat']);
```

Comment:
Have a look at the logic in [Raptor's log stream](https://github.com/mozilla-raptor/raptor-cli/blob/master/lib/device/lib/logging.js#L170). Without determining a serial, we could get the wrong log for multiple connected devices. Maybe the endpoints should be targeted to a serial, e.g. `GET /logcat/<serial>`?